styles/layout/_topnav.scss

```diff
@@ -17,6 +23,12 @@
   .logo-wrapper,
   .topnav-right-container {
     @include span(4);
+    position: relative;
+    height: $topnav-height;
+
+    > ul {
+      @include position(absolute, null 0 0 0);
+    }
 
     > ul > li {
       @include span(6);
```

Comment:
This is incorrect.  Each ul > li is span(2), which, at full-res, is a 10px width difference from what yours end up as.